ovos_date_parser/__init__.py

```
@@ -286,3 +292,311 @@ def extract_datetime(
+        if date_format == 'MDY':
+            return "{} {}".format(month, dt.strftime("%d"))
+        else:
+            return "{} {}".format(dt.strftime("%d"), month)
+    return dt.strftime("%d")
+
+
+def nice_weekday(dt, lang):
+    full_code = standardize_lang_tag(lang)
+    date_time_format.cache(full_code)
+
+    if full_code in date_time_format.lang_config.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)